config/vehicle_info.param.yaml

```diff
@@ -1,12 +1,12 @@
 /**:
   ros__parameters:
     wheel_radius: 0.39
```

Comment:
No need to update this to `0.383` or `0.38`? (766/2)
https://cars-japan.net/tire/tire_gaikei.cgi?act=view&zsize=235/55R20

Reply:
Or the old model has a bit larger size?

Reply:
I fotgot it!
I'll change wheel_radius to 0.38 (20inch)

Reply:
I've fixed at [b8ee382](https://github.com/tier4/lexus_description.iv/pull/35/commits/b8ee382ddfd26d236fd4b209b9359684508c1114)